src/Features/VisualBasic/Portable/ExtractMethod/VisualBasicSelectionValidator.vb

```diff
@@ -494,6 +494,18 @@ result.ReadOutside().Any(Function(s) Equals(s, local)) Then
                        }
             End If
 
+            Dim commonRootContainer = commonRoot.FirstAncestorOrSelf(Of SyntaxNode)(Function(node) node.IsKind(SyntaxKind.ClassBlock) Or node.IsKind(SyntaxKind.StructureBlock) Or node.IsKind(SyntaxKind.ModuleBlock))
+
+            If commonRootContainer Is Nothing Then
+                Return New SelectionInfo With
+                    {
+                        .Status = New OperationStatus(OperationStatusFlag.None, VBFeaturesResources.No_valid_selection_to_perform_extraction),
```

Comment:
this string seems to be duplciated across languages for some reason.  can you move into FeaturesResources and remove from C# and VB.